Update ont-zte-f6005.md (#194)
hack-gpon/hack-gpon.github.io: _ont/ont-zte-f6005.md

# Usage

{% include_relative ont-nokia-use.md %}

{% include_relative ont-nokia-useful-command.md %}

## Enable password

{% include alert.html content="The following enable password is used to enter GponCLI over Serial, but currently in the models distributed in Italy by TIM and OpenFiber there is no way to enter GponCLI by Serial, only by Telnet after flashing custom firmware. The enable password is not useful for entering the Web Gui." alert="Note" icon="svg-info" color="blue" %}

You can use this tool to generate the enable password:

{% include cig_password.html username="ont" %}

# Advanced settings
## Unlock serial and TELNET

> This ONT is the twin brother of **CIG G97-CP**, if you can find its bootloader (named `CIG_bu.en_V3.09.15`), you can easly repack firmware and enable TTL. 

Reviewer: ```suggestion
This ONT is the twin brother of [CIG G97-CP](/ont-cig-g-97cp), if you can find its bootloader (named `CIG_bu.en_V3.09.15`), you can easly repack firmware and enable TTL. 
```

> This ONT is the twin brother of [CIG G97-CP](/ont-cig-g-97cp), if you can find its bootloader (named `CIG_bu.en_V3.09.15`), you can easly repack firmware and enable TTL. 

Reviewer 2: ```suggestion
This ONT is the twin brother of [CIG G97-CP](/ont-cig-g-97cp), if you can find its bootloader (named `CIG_bu.en_V3.09.15`), you can easly repack the firmware and enable its serial port. 
```

> You need a SPI programmer (that should be a 3.3V, like modded CH341a) to read and write back the flash.

Author: uhm.. not all CH341a support 3.3V, so you need to mod it or get one that have switch from 5V to 3.3V

> You need a SPI programmer (that should be a 3.3V, like modded CH341a) to read and write back the flash.

Reviewer 2: ```suggestion
You need a 3.3V SPI programmer (like a modded CH341a) to read and write back the flash.
```

{% include alert.html content="This was tested only on TIM V6.0.10N20 firmware!" alert="Note" icon="svg-info" color="blue" %}

> After you got a full dump of your ONT here is the procedure to cut&paste new bootloader and have TTL enabled:

Reviewer 2: ```suggestion
After you got a full dump of your ONT, here is the procedure to replace the original bootloader with the CIG one enable the serial port:
```

Cut old bootloader:
```sh
dd if=flash_dump.bin of=no_boot_flash_dump.bin bs=1 skip=184064
```

> Attach new one:

Reviewer 2: ```suggestion
Attach the new one:
```

```sh
cat CIG_bu.en_V3.09.15 no_boot_flash_dump.bin > mod-boot_flash_dump.bin
```

Now you can flash the file `mod-boot_flash_dump.bin` back to your SPI. 

> After power-up the ONT, the TTL will print this message:

Reviewer 2: ```suggestion
After powering up the ONT, the serial port will print this message:
```

```sh
**************************************
*                                    *
*  KEY -- Enter console terminal     *
*                                    *
**************************************
```

To access full U-Boot a special escape sequence is needed. 

If you use TeraTerm software, create a simple MACRO file that contains this code:

`send $1B $1D $0F $0B`

> Configure TeraTerm with the correct serial parameters (refer to **Serial** paragraph), select the created MACRO file **BEFORE** power-up the ONT but **DON'T OPEN IT NOW**, power-up the ONT and when you see the above message, quickly open the macro to reach U-Boot prompt:

Reviewer 2: ```suggestion
Configure TeraTerm with the correct serial parameters (refer to **Serial** paragraph), select the created MACRO file **BEFORE** powering up the ONT but **DON'T OPEN IT NOW**, power-up the ONT and when you see the above message, quickly open the macro to reach the U-Boot prompt:
```

```sh
9601D#
```

{% include alert.html content="Note that this proceedure needs to be done each time you power-cycle the ONT" alert="Note" icon="svg-info" color="blue" %}

Now with the U-Boot prompt a custom firmware that enable TELNET can be flashed. 
Please note that if you use a **TIM** or **OpenFiber** base firmware, the TTL will be silent after kernel loading because it was disabled at kernel level.

> Here is the procedure to flash a custom firmware on your ONT thru U-Boot console:

Reviewer 2: ```suggestion
Here is the procedure to flash a custom firmware on your ONT via the U-Boot console:
```

# Usage

{% include_relative ont-nokia-use.md %}

{% include_relative ont-nokia-useful-command.md %}

## Enable password

> {% include alert.html content="The following enable password is used to enter GponCLI, but currently in the models distributed in Italy by TIM and OpenFiber there is no way to enter GponCLI either by SSH, Telnet or Serial. The enable password is not useful for entering the Web Gui." alert="Note" icon="svg-info" color="blue" %}

Reviewer: Edit also this alert

{% include cig_password.html username="ont" %}

# Advanced settings
## Unlock serial and TELNET

This ONT is the twin brother of **CIG G97-CP**, if you can find its bootloader (named `CIG_bu.en_V3.09.15`), you can easly repack firmware and enable TTL. 
You need a SPI programmer (that should be a 3.3V, like modded CH341a) to read and write back the flash.

> {% include alert.html content="This was tested on TIM V6.0.10N20 firmware!" alert="Note" icon="svg-info" color="blue" %}

Reviewer: ...Only on...

After you got a full dump of your ONT here is the procedure to cut&paste new bootloader and have TTL enabled:

Cut old bootloader:
```sh
dd if=flash_dump.bin of=no_boot_flash_dump.bin bs=1 skip=184064
```

Attach new one:
```sh
cat CIG_bu.en_V3.09.15 no_boot_flash_dump.bin > mod-boot_flash_dump.bin
```

Now you can flash the file `mod-boot_flash_dump.bin` back to your SPI. 

After power-up the ONT, the TTL will print this message:

```sh
**************************************
*                                    *
*  KEY -- Enter console terminal     *
*                                    *
**************************************
```

To access full U-Boot a special escape sequence is needed. 

If you use TeraTerm software, create a simple MACRO file that contains this code:

> `send $1B $1D $0F $0B`

Reviewer: Didn't you talk about it changing by version?

Author: what do you mean?

Configure TeraTerm with the correct serial parameters (refer to **Serial** paragraph), select the created MACRO file **BEFORE** power-up the ONT but **DON'T OPEN IT NOW**, power-up the ONT and when you see the above message, quickly open the macro to reach U-Boot prompt:

```sh
9601D#
```

> **_Note that this proceedure needs to be done each time you power-cycle the ONT_**

Reviewer: Use alert
